server/services/omuCaselist.test.ts

```diff
@@ -0,0 +1,187 @@
+import { add, sub, format } from 'date-fns'
+import OmuCaselist from './omuCaselist'
+import LicenceStatus from '../enumeration/licenceStatus'
+import LicenceType from '../enumeration/licenceType'
+import Container from './container'
+
+describe('omu caselist', () => {
+  it('should return prison view case', () => {
+    const today = format(new Date(), 'yyyy-MM-dd')
+    const omuCaselist = createCaselist(LicenceStatus.SUBMITTED, today, '2022-10-19')
+
+    expect(omuCaselist.getPrisonView()).toEqual({
+      excludedItems: [],
+      items: [
+        {
+          deliusRecord: {
```

Comment:
for this you don't really care about these details:
```
 expect(omuCaselist.getPrisonView().case).toHaveLength(1)
 expect(omuCaselist.getPrisonView().excluded).toHaveLength(0)

Reply:
done